source/h2_connection.c

```diff
@@ -216,6 +234,20 @@ static struct aws_h2_connection *s_connection_new(
         goto error;
     }
 
+    if (aws_hash_table_init(
+            &connection->thread_data.closed_streams_where_frames_might_trickle_in,
```

Comment:
Yeah, I purposefully made this waaaay too explicit because it's so stupidly nuanced.

`recently_closed_streams` seemed like a good name until I realized that I didn't need to store the IDs of streams that closed in certain ways.